charts/node-server/templates/mongodb-deploy.yaml

```
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mongodb
+  labels:
+    app: mongodb
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: mongodb
+  template:
+    metadata:
+      labels:
+        app: mongodb
+    spec:
+      containers:
+        - name: mongodb
+          image: mongo:5.0.11
+          args:
+            - --dbpath
+            - /tl/data/db
+          lifecycle:
+            postStart:
+              exec:
+                command:
+                  - sh
+                  - -c
+                  - |
+                    mkdir -p /tl/data/db
+                    chmod 777 /tl/data/db
```

It seems to me that creating the directory structure inside the lifecycle may lead to troubles in some cases as [it runs asynchronously](https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/#container-hooks) along with container entrypoint
The way I see people are doing this stuff is with a script when ends with `exec`, you can add the script with the config map to keep things organised, (see other comment)
```suggestion
          command:
            - 'scripts/entrypoint.sh'
```